Formula/octave.rb

```diff
@@ -96,6 +96,18 @@ def install
                           "--with-sndfile"
     system "make", "all"
 
+    on_linux do
+      # Explicitly specify aclocal and automake without versions
+      args << "ACLOCAL=aclocal"
+      args << "AUTOMAKE=automake"
+
+      # Mesa OpenGL location must be supplied by LDFLAGS on Linux
+      args << "LDFLAGS=-L#{Formula["mesa"].opt_lib} -L#{Formula["mesa-glu"].opt_lib}"
```

Comment:
Is this a recursive dep on Linux?

Reply:
What do you mean?

Reply:
There's no `depends_on "mesa"` or `depends_on "mesa-glu"` in the formula, so if it's not a recursive dependency then I'm not sure how this line changes anything.

Reply:
https://github.com/Homebrew/linuxbrew-core/blob/75ccb46aa31a654b310cbea41fb750ff8a666a96/Formula/qt%405.rb#L42

Reply:
Yeah it comes through qt/qt5. I did not really respect the order here, I'll work on a qt PR later on.